BUILDING.md

```diff
@@ -71,8 +71,9 @@ There are three support tiers:
 
 For production applications, run Node.js on supported platforms only.
 
-Note that end-of-life (EoL) platforms are not supported even if listed below
-and can be removed at any moment.
+End-of-life (EoL) platforms are not supported even if listed below and can be
+removed at any moment. The community does not build or test against EoL
```

Comment:
I'd be slightly concerned that this could be read as that we "can't" even if it's what we think need to be the case for some period of time.  How about "The community avoids building or testing against EoL platforms"

Reply:
```suggestion
removed at any time. The community does not build or test against EoL
```